chain/chain/src/store.rs

```diff
@@ -1998,6 +2017,10 @@ impl<'a> ChainStoreUpdate<'a> {
             self.gc_col(ColNextBlockWithNewChunk, &height_shard_id);
             let key = StateHeaderKey(shard_id, block_hash).try_to_vec()?;
             self.gc_col(ColStateHeaders, &key);
+
+            if self.get_last_block_with_new_chunk(shard_id)? == Some(&block_hash) {
+                self.gc_col(ColLastBlockWithNewChunk, &index_to_bytes(shard_id));
+            }
```

Comment:
Are there any negative consequences of not GCing this column?
Let's only add GCing now for columns that can potentially result in memory consumption growing uncontrollably.

Reply:
> Are there any negative consequences of not GCing this column?

Let's say it will not cleared and point to deleted Block after complete Sync. Did we consider this case while designing the column?

I guess it may be more useful to keep Storage in consistent state rather than have non-cleared columns specifically.